src/olympia/activity/tests/test_serializers.py

```diff
@@ -94,7 +94,7 @@ def test_sanitized_activity_detail_not_exposed_to_developer(self):
         # Comments should be the santized text rather than the actual content.
         assert result['comments'] == amo.LOG.REQUEST_SUPER_REVIEW.sanitize
         assert result['comments'].startswith(
-            'The addon has been flagged for Admin Review.')
+            'The addon has been flagged for Admin Rating.')
```

Comment:
This is a reviewer review :)  - Admin Review being the alternate name for Super Review

Reply:
woops, overzealous search&replace, I kinda assume that all instances of `Review` with the capital R were references to the model...